pom.xml

```diff
@@ -177,26 +217,10 @@
                             </gpgArguments>
                         </configuration>
                     </plugin>
-                    <plugin>
-                        <groupId>org.springframework.boot</groupId>
-                        <artifactId>spring-boot-maven-plugin</artifactId>
-                        <version>2.4.12</version>
-                        <executions>
-                            <execution>
-                                <id>repackage</id>
-                                <goals>
-                                    <goal>repackage</goal>
-                                </goals>
-                                <configuration>
-                                    <skip>true</skip>
```

Comment:
Because this execution is skipped, I don't see how this plugin would be doing anything. I did a build with and without the plugin enabled and didn't see any changes in files.